src/components/Tokens/TokenDetails/PopoverText.tsx

```diff
@@ -0,0 +1,36 @@
+import { Placement } from '@popperjs/core'
```

Comment:
Could you explain to me the usage of `popperjs` in the project? I've searched for all its occurrences and so far, I've only found it being used in 2 files as a type only import! Do we use it somewhere, e.g. to power a popover? If yes, where is that line/file located?

Reply:
Also, nit, you can use `import type` when importing types only, that way, the whole import gets stripped down while transpiling to Javascript! It is also a great way to import types from dev dependencies. But this is additional use case beyond this comment.

Reply:
Thanks for the review! & for the tip abt importing types. I ended up deleting these components as I realized we could just update the existing tooltip component to reflect the latest UI theme, rather than wrapping it.

From what I know, popperjs is a library that handles creating a popover ui element that intuitively moves based on the size of the screen / positioning of other elements. we use this in src/components/Popover which is then used by src/components/Tooltip